insights.staging.ubuntu.com/README.md

```diff
@@ -0,0 +1,20 @@
+This deployment can be released file by file but it can also be released in one by pointing kubectl at the parent folder.
```

Comment:
This file & dir should no longer exist. Please rebase off master.

Reply:
It was already rebased, this was in a commit for some reason. This should be corrected now, thanks.